nUpdate.Administration/UI/Dialogs/ProjectEditDialog.cs

```diff
@@ -994,9 +994,13 @@ private void continueButton_Click(object sender, EventArgs e)
                     _useStatistics = useStatisticsServerRadioButton.Checked;
```

Comment:
Sorry, I have not much experience. So I'm puzzled. I don't know how I need to change the "submit title". Can you give me a reference or convention description?

Reply:
The commit messages should be named so they fit in with the sentence "When applied, this commit will..."

So, it is only "Fixed #48 and another same bug" that does not correspond to the criteria. This would result in "When applied, this commit will fixed #48..." which is not correct. You understand, what I mean? So it should just be "Fix" instead of "Fixed". Then it is fine.